test_main.py

```diff
@@ -34,7 +38,7 @@ def test_get_books(self, test_db):
         book2 = create_book(test_db, BookIn(**TEST_BOOKS[1]))
```

Comment:
```suggestion
```suggestion
        # Clear any pre-existing data to ensure a clean slate
        test_db.query(models.Book).delete()
        test_db.commit()
```
```